desc/compute/_field.py

```diff
@@ -193,19 +200,35 @@ def _B0_r(params, transforms, profiles, data, **kwargs):
     label="\\partial_{\\rho} B^{\\theta}",
     units="T \\cdot m^{-1}",
     units_long="Tesla / meter",
-    description="Contravariant poloidal component of magnetic field, derivative wrt "
-    + "radial coordinate",
+    description=(
+        "Contravariant poloidal component of magnetic field, derivative wrt radial"
+        " coordinate"
+    ),
     dim=1,
     params=[],
     transforms={},
     profiles=[],
     coordinates="rtz",
-    data=["B0", "B0_r", "iota", "iota_r", "lambda_z", "lambda_rz"],
+    data=[
+        "B0",
+        "B0_r",
+        "iota",
+        "iota_r",
+        "lambda_rz",
+        "lambda_z",
+        "omega_rz",
+        "omega_z",
+    ],
 )
 def _B_sup_theta_r(params, transforms, profiles, data, **kwargs):
-    data["B^theta_r"] = data["B0_r"] * (data["iota"] - data["lambda_z"]) + data[
-        "B0"
-    ] * (data["iota_r"] - data["lambda_rz"])
+    data["B^theta_r"] = data["B0"] * (
```

Comment:
To clarify,in general I needed to rearrange the expressions of some quantities to
1. remove a division by 0
2. recast into a big fraction numerator / denominator to compute the limit
I don't think I did that here though.

It might be easier if we make a new PR with just the changes `_basis_vectors.py` and `_core.py` and merge that in first. Then you should be able to use GitHub to diff the PR against your `rc/toroidal_angle` branch instead of master.